Standards/scs-0402-v1-status-page-openapi-spec-decision.md

```diff
@@ -82,6 +82,46 @@ An impact defines the relation between an incident and a component. A component
 
 To reflect this, each component and incident can have a list of impacts, stating the type of impact and a reference to the incident or component, it refers to.
 
+Furthermore a severity value is added to the impact when referencing an component, to gauge the impacts severity on the system.
```

Comment:
Couple typos. Fixed: 

``Furthermore, a _severity value_ is added to the `Impact` when referencing a component, to gauge the impact's severity on that component.``